pkgs/development/lua-modules/overrides.nix

```diff
@@ -270,6 +270,13 @@ with prev;
   });
 
   lua-iconv = prev.lua-iconv.overrideAttrs (oa: {
+    # rockspec is broken https://github.com/ittner/lua-iconv/issues/8
+    src = fetchFromGitHub {
```

Comment:
This workaround should be obsolete, you can go back to just using the default LuaRock.

Reply:
indeed, I removed it